scripts/ssl/ca-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Hmm....
I just went through security training.
I guess that putting it in the repo may validate our [best practices](https://memsql.atlassian.net/wiki/spaces/INFORMATIO/pages/3577708545/Secrets+Management+Guidelines#My-secret-is-not-relevant-for-production%2C-it%E2%80%99s-just-for-testing%2Fdevelopment%2Fresearch.). 
Is it possible to move these files to the CI/CD?
Optionally, we can encode it, and put credentials for decoding in CI/CD